packages/auth/e2e/helpers.js

```diff
@@ -212,7 +212,7 @@ exports.createUserWithMultiFactor = async function createUserWithMultiFactor() {
   const email = 'verified@example.com';
   const password = 'test123';
   await createVerifiedUser(email, password);
-  const multiFactorUser = await firebase.auth.multiFactor(firebase.auth());
+  const multiFactorUser = await firebase.auth().multiFactor(firebase.auth().currentUser);
```

Comment:
If we want to follow the API closely, then the `multiFactor` function should be available as import and not be derived from an Auth instance. Looking at the implementation in `firebase-js-sdk` we could probably copy that (packages/auth/src/mfa/mfa_user.ts): Instantiate the user by copying the data from the user object.

Reply:
again I think this is just an artifact of me taking a v9-only JS API and faking a "v8 style" for it. This should resolve exactly as you say in the coming v9 style - it will be an export and you'll be sending result of `getAuth()` into it

Reply:
👍